CodeEditModules/Package.swift

```diff
@@ -72,9 +72,9 @@ let package = Package(
         ),
         .package(
             name: "Preferences",
-            url: "https://github.com/sindresorhus/Preferences.git",
-            from: "2.5.0"
-        ),
+            url: "https://github.com/underthestars-zhy/Preferences.git",
```

Comment:
Why?

Reply:
Because my package has a feature that can know the window is focused or not.

`@StateObject var model = KeyModel.shared`
```swift
if model.key {
                        Color.focusedColor.opacity(appearance == .system || appearance == .default ? 1 : 0)
                            .frame(width: 67 + 3, height: 46 + 3)
                            .cornerRadius(5)
                    } else {
                        Color.unfocusedColor.opacity(appearance == .system || appearance == .default ? 1 : 0)
                            .frame(width: 67 + 3, height: 46 + 3)
                            .cornerRadius(5)
                    }
```